lib/api/controllers/documentController.js

```diff
@@ -140,17 +168,17 @@ function DocumentController (kuzzle) {
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.mCreate = function documentMCreate (request) {
-    return doMultipleWriteActions(kuzzle, request, 'create');
-  };
+  mCreate(request) {
+    return doMultipleWriteActions(this.kuzzle, request, 'create');
+  }
 
   /**
    * Create a new document through the persistent layer. If it already exists, update it instead.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 97 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)